js/correctness/no-replaceAll.yaml

```diff
@@ -0,0 +1,11 @@
+rules:
+- id: no-replaceall-javascript
+  patterns:
+  - pattern: $STRING.replaceAll("...",$NEW)
+  - pattern: $STRING.replaceAll($OLD, $NEW)
```

Comment:
you can just take out the second pattern, remove the autofix, and looks like it'll be good to go.

Reply:
thanks, fixed! Can you approve? Should I add it to any rulesets or is putting it here enough?